posthog/temporal/batch_exports/s3_batch_export.py

```diff
@@ -360,6 +361,25 @@ async def initialize_and_resume_multipart_upload(inputs: S3InsertInputs) -> tupl
     return s3_upload, interval_start
 
 
+def s3_default_fields() -> list[BatchExportField]:
```

Comment:
Ah, this is kinda doing what I called out in the other PR. But I guess if they're actually that specific to each backend then maybe there won't be much reuse anyway.

Reply:
Yeah, each backend is a bit specific, but I can take a second look in case there is some overlap here on there.